packages/popover/src/Popover.tsx

```diff
@@ -21,13 +22,17 @@ export const Popover: React.FC<IPopover> = ({
   onVisibleChange,
   withArrow = true,
   shouldFitContainer = false,
+  observeContentSize = false,
```

Comment:
autoContentSize - will be correct, like autoFlip. observe - to watch carefully the way something happens but this does not mean that something will change